scripts/create-change-file.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+ 
+let fs = require('fs');
+let glob = require('glob')
+let guid = require('guid');
+let inquirer = require('inquirer');
+let path = require('path')
+
+let packages = [];  // keeps track of package.json files that have been found
+let answers = {};
+let data = [];
+
+// Gets all the package.json files under the current executing directory
+let getPackages = () => {
+    glob('**!(node_modules)/package.json', /*options*/ {}, (error, files) => {
+        packages = files.forEach((file) => {
+            packages.push(path.basename(path.dirname(file)));   // push the package name of the parent directory
+        });;
+        promptUser();
+    });
+}
+
+// Questions related to the project. Had to split into two sets of questions in case user selects additional help
+let projectQuestions = [
+    {
+      name: 'projectName',
+      type: 'list',
+      message: 'Select the project you would like to bump:',
+      choices: packages
+    },
+    {
+        name: 'bumpType',
+        type: 'list',
+        message: 'Select the type of change:',
+        default: 'patch',
+        choices: [
+            'major - for breaking changes (ex: renaming a file)',
+            'minor - for adding new features (ex: exposing a new public API)',
+            'patch - for fixes (ex: updating how an API works w/o touching its signature)',
+            'help'
+        ]
+    }
+];
+
+// Questions related to the change
+let changeQuestions = [
+    {
+        name: 'comments',
+        type: 'input',
+        message: 'Please add any comments to help explain this change:'
+    },
+    {
+        name: 'addMore',
+        type: 'confirm',
+        message: 'Would you like to bump any additional projects?'
```

Comment:
How about the scenario where you want to log multiple changes (multiple lines of comments) into the same file? Can you change this message to say bump any additional projects, or add additional change messages to the existing bump? 

Perhaps we should explain that you can identify "minor" twice but it will only bump once.. Wonder if anyone would fear creating 2 minor entries due to concern of having minor bumped twice...